index.html

```diff
@@ -0,0 +1,155 @@
+       </form>
+      </section>
+<!-- services Sections -->
+      <section id="services">
+        <div class="items">
+          <i class="fa-solid fa-fire"></i>
+          <div class="text">
+            <h3>Premium Materials</h3>
+          <p>Our trombones use the shiniest brass which is sourced locally. 
+            This will increase the longevity of your purchase.</p>
+          </div>
+        </div>
+    
+
+
+        <div class="items">
+          <i class="fa-solid fa-truck"></i>
+          <div class="text">
+           <h3>Fast Shipping</h3>
+          <p>We make sure you recieve your trombone as soon as we have 
+            finished making it. We also provide free returns if you are not satisfied.</p>
+          </div>
+        </div>
+
+
+        <div class="items">
+          <i class="fa-solid fa-battery-full"></i>
+          <div class="text">
+            <h3>Quality Assurance</h3>
+            <p>For every purchase you make, we will ensure there are no 
+              damages or faults and we will check and test the pitch of 
+              your instrument.</p>
+          </div>
+        </div>
+
+      </section>
+
+      <section class="video">
+        <iframe src="https://www.youtube.com/embed/tgbNymZ7vqY" id="video">
+        </iframe>
+      </section>
+      <!-- price seection -->
+      <section id="price">
+        <div class="price-item">
+          <div class="top">
+            <h4>Tenor Trombone</h4>
+          </div>
+          <div class="price">$600</div>
+          <div class="text">
+            <p>Lorem, ipsum dolor.</p>
+            <p>Lorem ipsum dolor sit amet.</p>
+            <p>Lorem ipsum dolor sit.</p>
+            <p>Lorem, ipsum dolor.</p>
+          </div>
+          <button>Select</button>
+        </div>
+
+
+        <div class="price-item">
+          <div class="top">
+            <h4>Bass Trombone</h4>
+          </div>
+          <div class="price">$900</div>
+          <div class="text">
+            <p>Lorem, ipsum dolor.</p>
+            <p>Lorem ipsum dolor sit amet.</p>
+            <p>Lorem ipsum dolor sit.</p>
+            <p>Lorem, ipsum dolor.</p>
+          </div>
+          <button>Select</button>
+        </div>
+
+
+        <div class="price-item">
+          <div class="top">
+            <h4>Valve Trombone</h4>
+          </div>
+          <div class="price">$1200</div>
+          <div class="text">
+            <p>Plays similar to a Trumpet</p>
+            <p>Great for Jazz Bands</p>
+            <p>Lorem ipsum dolor.</p>
+            <p>Lorem ipsum.</p>
+          </div>
+          <button>Select</button>
+        </div>
+      </section>
+    </main>
+
+    <footer>
+      <div class="container">
+        <ul class="links">
+          <li><a href="#">Privacy</a></li>
+          <li><a href="#">Terms</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+        <p>Copyright 2022, Original Trombones</p>
+      </div>
+    </footer>
+  
+   
+    
+
+
+
+
```

Comment:
I recommend that you remove extra spaces on your project. Removing it will makes your codes easier to read and makes it more professional.